### ISSUE TITLE
✨ Set up Copilot instructions

### ISSUE THREAD

**Issue by Alpha48Alpha** [Alpha48Alpha/Ai-montizable-#18]
git checkout -b my-new-branch git clone https://github.com/your-org/project-name.git
cd project-name
pnpm install
pnpm dev
Generate a complete Python reinforcement learning research lab called Alpha48Alpha AI Lab.

Requirements:
- Use Python and PyTorch
- Modular architecture for experimentation
- Support training RL agents in simulated environments

Project structure:

alpha48alpha_ai_lab/
    README.md
    requirements.txt
    train.py
    config.py

    agents/
        policy_gradient_agent.py
        dqn_agent.py

    environments/
        grid_world.py
        world_simulator.py

    models/
        neural_policy.py
        value_network.py

    training/
        trainer.py
        replay_buffer.py

    utils/
        logger.py
        visualization.py

Features:
- grid world simulation environment
- reinforcement learning training loop
- policy gradient implementation
- deep Q-learning implementation
- reward tracking and logging
- visualization of agent learning

Include detailed comments and clean, production-style code.
Create a world simulation environment in Python for reinforcement learning.

Environment features:
- 2D world
- agent navigation
- obstacles
- goal states
- rewards and penalties

Functions required:
- reset()
- step(action)
- render()
- get_state()
Create a reinforcement learning training loop using PyTorch.

Requirements:
- train agent in simulated environment
- collect episodes
- compute rewards
- update policy network
- log training metrics

Use policy gradient optimization.

The environment should be compatible with reinforcement learning training loops.mkdir alpha48alpha_ai_lab
cd alpha48alpha_ai_lab
python -m venv venv
source venv/bin/activate
pip install torch numpy matplotlib
code .python train.py
